### PR TITLE
Feature/postgres adapter

### PR DESCRIPTION
**Pull request by GopherJ**

### CODE REVIEW

File: Cargo.toml [hunk 1]
```diff
@@ -10,3 +10,9 @@ edition = "2018"
 regex = "1"
 rhai = "0.7"
 ip_network = "0.3.4"
+diesel = { version = "1.4.2", features = ["r2d2", "postgres"], optional = true }
+
+[features]
+default = ["postgres"]
```
Comment (xcaptain) [inline Cargo.toml]: How about we put this adapter to another workspace, so we can publish as 2 separate package to crate.io

Comment (GopherJ) [reply Cargo.toml]: yes that will be a good idea, maybe later we can seperate adapters into workspaces

**Comment by GopherJ** [comment]
@hsluoyz could you check if some changes are needed? currently user can disable this default feature by adding this in `Cargo.toml`
```
[dependencies.A]
version = "*"
default-features = false
```

**Comment by hsluoyz** [comment]
@GopherJ @xcaptain , all the DB adapters should be put in another repo, not this main library. I suggest putting code to: https://github.com/casbin-rs/diesel-adapter

**Comment by GopherJ** [comment]
@hsluoyz ok, I also think I should make it supports `mysql`, `postgresql`, `sqlite` together because they are supported by `diesel`. I'll do that later

**Comment by hsluoyz** [comment]
Sounds reasonable. I will close this PR for now.

**Pull request closed without merging**